src/css/styles.scss

```diff
@@ -51,65 +51,80 @@ article {
   break-inside: avoid;
```

Comment:
I have never heard of break-inside: avoid, might need you to explain this later @RayRedGoose